processes/troubleshooting/README.MD

```diff
@@ -1,21 +0,0 @@
-# Production Troubleshooting Processes
-
```

Comment:
The general purpose fallback process documented here isn't useful because it doesn't include technical steps for performing these actions. This type of doc should live in the playbook (which could also use some love, but out of scope here).